### PR TITLE
bump Android version

### PR DESCRIPTION
**Pull request by nickpatrick**

### CODE REVIEW

File: ios/RNRadar.m [hunk 1]
```diff
@@ -91,7 +91,7 @@ - (void)didLogMessage:(NSString *)message {
     }
 }
 
-RCT_EXPORT_METHOD(initialize:(NSString *)publishableKey) {
+RCT_EXPORT_METHOD(initialize:(NSString *)publishableKey fraud:(BOOL)fraud) {
```
Comment (nickpatrick) [inline ios/RNRadar.m:94]: no iOS equivalent

File: android/src/main/java/io/radar/react/RNRadarModule.java [hunk 2]
```diff
@@ -78,9 +78,13 @@ public String getName() {
     }
 
     @ReactMethod
-    public void initialize(String publishableKey) {
-        Radar.initialize(getReactApplicationContext(), publishableKey);
-        Radar.setReceiver(receiver);
+    public void initialize(String publishableKey, boolean fraud) {
+        if (fraud) {
```
Comment (nickpatrick) [inline android/src/main/java/io/radar/react/RNRadarModule.java:82]: only changes `initialize()` logic if `fraud == true`

File: android/src/main/java/io/radar/react/RNRadarModule.java [hunk 3]
```diff
@@ -78,9 +78,13 @@ public String getName() {
...
+            Radar.initialize(getReactApplicationContext(), publishableKey, receiver, Radar.RadarLocationServicesProvider.GOOGLE, fraud);
+        } else {
+            Radar.initialize(getReactApplicationContext(), publishableKey);
+            Radar.setReceiver(receiver);
```
Comment (lmeier) [inline android/src/main/java/io/radar/react/RNRadarModule.java:86]: Hmm, unless I'm missing something, I think we want to set the receiver regardless of whether or not `fraud: true`

Comment (nickpatrick) [reply android/src/main/java/io/radar/react/RNRadarModule.java:86]: I could remove the `if/else` and collapse this to `Radar.initialize(getReactApplicationContext(), publishableKey, receiver, Radar.RadarLocationServicesProvider.GOOGLE, fraud)`. Just didn't want to touch existing `initialize()` logic if not necessary.

File: example/App.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ export default function App() {
   const stringify = (obj) => JSON.stringify(obj, null, 2);
 
   useEffect(() => {
-    Radar.initialize("prj_test_pk_4ce462a534c0f749af3fdaac6fff9f485d65e720");
+    Radar.initialize("prj_test_pk_4ce462a534c0f749af3fdaac6fff9f485d65e720", true);
```
Comment (lmeier) [inline example/App.js:42]: Let's change this key back to `prj_test_pk_0000000000000000000000000000000000000000`. A test key accidentally made it here (now invalidated).

**Review by lmeier — APPROVED** [review]
noted 2 small things. apart from those, if it's building correctly and testing well for you, this is gtg 🚀

File: android/src/main/java/io/radar/react/RNRadarModule.java (hunk 3 above)
Comment (lmeier) [reply android/src/main/java/io/radar/react/RNRadarModule.java:86]: Way of doing that while also setting the receiver if fraud is true would be:
```
if (fraud) {
    Radar.initialize(getReactApplicationContext(), publishableKey, receiver, 
    Radar.RadarLocationServicesProvider.GOOGLE, fraud);
Radar.setReceiver(receiver);
} else { 
    Radar.initialize(getReactApplicationContext(), publishableKey);
    Radar.setReceiver(receiver);
}

File: package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "description": "React Native module for Radar, the leading geofencing and location tracking platform",
   "homepage": "https://radar.com",
   "license": "Apache-2.0",
-  "version": "3.8.4",
+  "version": "3.8.5",
```
Comment (tjulien) [inline package.json:6]: `npm install` to update package-lock.json

File: android/build.gradle [hunk 6]
```diff
@@ -45,5 +45,5 @@ repositories {
 
 dependencies {
     api 'com.facebook.react:react-native:+'
-    api 'io.radar:sdk:3.8.5'
+    api 'io.radar:sdk:3.8.6'
```
Comment (tjulien) [inline android/build.gradle:48]: above: `versionName '3.8.5'`

File: android/src/main/java/io/radar/react/RNRadarModule.java (hunk 3 above)
Comment (nickpatrick) [reply android/src/main/java/io/radar/react/RNRadarModule.java:86]: Do we think the separate `setReceiver()` call is needed if `receiver` is passed to `initialize()`?

File: package.json (hunk 5 above)
Comment (nickpatrick) [reply package.json:6]: Done

File: android/src/main/java/io/radar/react/RNRadarModule.java (hunk 3 above)
Comment (nickpatrick) [reply android/src/main/java/io/radar/react/RNRadarModule.java:86]: Cool

**Pull request merged**